src/configure/configure.ts

```diff
@@ -280,15 +280,14 @@ class Orchestrator {
 
     private async analyzeNode(node: any): Promise<void> {
```

Comment:
Can we have a type for folder node as well? VS Code must have defined a type for that. We should use that if possible.

If so, then we can defined the argument type here as `IResourceNode | FolderNode`. (replace`FolderNode` with the type for folder)